swagger.yaml

```diff
@@ -56,24 +67,52 @@ paths:
         200:
           description: OK
           schema:
-            $ref: "#/definitions/SearchResponse"
+            $ref: "#/definitions/DepartmentResponse"
         400:
           description: Query term not specified
         500:
           description: Internal server error
 
-  /departments/search:
+  /health:
     get:
       tags:
-        - department
-      summary: "Department query API"
-      description: "Department query API which returns matching links to other departments"
+        - health
+      summary: "Health check API"
+      description: "API to query the app for current health status"
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/Health"
+        500:
+          description: Internal Server Error
```

Comment:
Consider rewriting this to match the `/health` endpoint spec for other apps (unless this one is done differently).

You should be able to use what is on the [`dp-cli` tool project generation template](https://github.com/ONSdigital/dp-cli/blob/55d527dabeff3680851b197c677df0cd213dea80/project_generation/content/templates/api/swagger.yaml.tmpl#L32)

Reply:
Believe this is on another ticket. Justin might be doing it.

Reply:
https://trello.com/c/O5KTMeJy/757-search-api-update-health-endpoint

Reply:
This will be looked at in the above ticket - thanks Noelle